content/docs/atom-guide/look-dev/tools/material-editor.md

```diff
@@ -0,0 +1,90 @@
+
+## Navigating Material Editor
+After Material Editor has opened and initialized it will display the main window, as shown below.
+
+![Material Editor](/images/atom-guide/tools/material-editor.jpg)
+
+For more information about features common to Atom tools, menus, dockable panels, and working with documents, see [Atom Tools Common Features](/docs/atom-guide/look-dev/tools/atom-tools-common-features/). 
+
+### Document Types and Views
+Material Editor supports creating, opening, and editing one type of document.
+
+#### Material Documents
+Material Documents are the main documents type in Material Editor.
+- ".material" files can be opened and edited.
+- ".materialtype" files can be opened but cannot be saved.
+- The material being edited by the active material document will be displayed on the model in the viewport. 
+- As changes are made to the material through the inspector or script, the material shown in the viewport will automatically updated to reflect the changes.
+
+## Main Menu
+The main menu contains all of the submenus and actions common to all Atom tools. 
+
+For more information about features common to Atom tools, menus, dockable panels, and working with documents, see [Atom Tools Common Features](/docs/atom-guide/look-dev/tools/atom-tools-common-features/). 
+
+## Viewport
+Use the **Viewport** panel to preview materials applied to models under different lighting conditions.
+
+{{< note >}}
+The Viewport cannot be closed entirely. If all the tabs are closed, the Viewport shows a default scene that contains a blank model, ground, and natural light. 
+{{< /note >}}
+
+For more information about the viewport and how to interact with it, see [Atom Tools Viewport](/docs/atom-guide/look-dev/tools/atom-tools-viewport/).  
+
+## Viewport Settings
+The Viewport Settings panel displays options to configure the Viewport properties. These properties control how the Viewport appears and which features are active. You can also create or edit configuration presets for the viewport model and the lighting environments. 
+
+For more information about the viewport settings, see [Atom Tools Viewport](/docs/atom-guide/look-dev/tools/atom-tools-viewport/).
+
+## Troubleshooting
+### Material Editor Fails To Launch
+Material Editor normally initializes all of the gems enabled by the active project. In order to reduce start times and system resource utilization, Material Editor, and the other atom tools, includes registry setting files that forcibly disable several standard O3DE gems that are not likely to be needed within the tool.
+
+If Material Editor fails to launch then it may be because of dependency issues with gems in the active project. Check MaterialEditor.log for any system entity or module initialization errors. If necessary, change or delete the custom registry settings from the Material Editor project registry folder.
```

Comment:
> Material Editor normally initializes all of the gems enabled by the active project

@gadams3  
I think this wording may be confusing for users. It sounds like Material Editor is the tool that initializes Gems in a project, which is confusing bc I would expect a more core O3DE system initializes them.  However, I think what this means is that Material Editor is dependent on several Gems, so in order to launch ME, it's dependencies have to be initialized as well. I assume these dependencies include Atom Core Gems that allow ME to function, as well as other Gems that may have material assets ME should be aware of. Does that sound right?

If so, consider the following rewording.  I try to lean away from speaking about the literal technical logic that occurs, and focus only on the details the user needs to troubleshoot. 

```suggestion
Material Editor initializes a set of Gems that allow it to either function properly or discover material assets. The set of Gems includes all of the Gems enabled by the active project, except the standard O3DE Gems that are certainly not needed within the tool. This optimization helps reduce start times and system resource utilization.

If Material Editor fails to launch, then it may be because of dependency issues with Gems in the active project. Check `MaterialEditor.log` for any system entity or module initialization errors. If necessary, change or delete the custom registry settings from the Material Editor project registry folder.
```

Reply:
To clarify, material editor, material canvas, etc require certain gems regardless of the project. All of the applications that come with O3DE automatically load every gem that the game project enables. There are several gems that we, the developers, know that material editor, material canvas etc, will never need. So, we identified them and block them from initializing. Launching material editor using the automated testing project without these settings would start/rub terrain, physics, emotion FX, an AWS client, script canvas, and much more

Reply:
This section should be the same for material editor and material canvas. Perhaps it should go into the common features page instead since it applies to all of the tools. This is really important to call out for material editor and material canvas because QA and others were once experiencing problems with it. The wording should mention that we load all of the project gems except for the ones disabled by these settings.